Userland/Libraries/LibWeb/Layout/TreeBuilder.cpp

```diff
@@ -21,14 +21,17 @@
 #include <LibWeb/HTML/HTMLButtonElement.h>
 #include <LibWeb/HTML/HTMLInputElement.h>
 #include <LibWeb/HTML/HTMLProgressElement.h>
+#include <LibWeb/HTML/HTMLTableCellElement.h>
```

Comment:
it seems this include is not needed anymore

Reply:
Indeed we don't, removed.